NebulaClient/PacketProcessors/Logistics/StationUIInitialSyncProcessor.cs

```diff
@@ -17,61 +18,59 @@ public class StationUIInitialSyncProcessor: IPacketProcessor<StationUIInitialSyn
     {
         public void ProcessPacket(StationUIInitialSync packet, NebulaConnection conn)
         {
-            StationComponent[] gStationPool = null;
-            if(packet.planetId == 0)
+            StationComponent stationComponent = null;
+            StationComponent[] gStationPool = GameMain.data.galacticTransport.stationPool;
+            StationComponent[] stationPool = GameMain.data.galaxy.PlanetById(packet.PlanetId).factory.transport.stationPool;
```

Comment:
Does this NRE when the client did not have the PlanetFactory loaded? (this is the case for planets he did not visit yet)

Reply:
This is when the client gets a response to opening a UIStationWindow, no?

So he must be on the planet. I could add ? before every period, but I don't see how it could happen.

Reply:
Ah yes you are right, my bad then :D

There should not be a way for a client to not have the factory loaded, even if he opens the ui and then leaves the planet before receiving this packet he still has the factory as he once was on the planet.